flake.nix

```
@@ -0,0 +1,18 @@
+{
+  description = "Deku development environment";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.esy-fhs.url = "github:d4hines/esy-fhs";
```

To de-duplicate nixpkgs you can use `follows` on `nixpkgs`
```suggestion
  inputs.esy-fhs.url = "github:d4hines/esy-fhs";
  inputs.esy-fhs.inputs.nixpkgs.follows = "nixpkgs";
```